### PR TITLE
Shop parser

### PR DESCRIPTION
**Pull request by Filari02**
beta version of ShopParser. started working with Senukai, but still need to expand logic (will do it soon). Do you have any thoughs about existing commits?

### CODE REVIEW

File: ShopParser/SenukaiParser.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            options.AddArguments("--headless");
+            _driver = new ChromeDriver();
+        }
+
+
+        public static void Main (string[] args)
+        {
+            var test = new SenukaiParser();
+            var tet = test.ParseShop();
+        }
```
Comment (Tomas-Per) [inline ShopParser/SenukaiParser.cs]: Don't think we need a main here, we will be running from WPF, this should be classified as a class library

**Review by Tomas-Per — CHANGES_REQUESTED** [review]
Main should be removed, this has to be a class library

File: ShopParser/SenukaiParser.cs (hunk 1 above)
Comment (Filari02) [reply ShopParser/SenukaiParser.cs]: removed Main

**Review by Tomas-Per — APPROVED** [review]

File: ShopParser/ShopParser.csproj [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```
Comment (Tomas-Per) [inline ShopParser/ShopParser.csproj]: Also change the output type to be a class library

Comment (Filari02) [reply ShopParser/ShopParser.csproj]: changed

**Pull request merged**